### PR TITLE
Create a new ES7 variable for forums.

### PR DESCRIPTION
**Pull request by dianakhuang**
Creates a new variable in ansible for forums. A corresponding PR in edx-internal is also being created.

### CODE REVIEW

File: playbooks/roles/forum/defaults/main.yml [hunk 1]
```diff
@@ -61,6 +63,7 @@ forum_base_env: &forum_base_env
   SINATRA_ENV: "{{ FORUM_SINATRA_ENV }}"
   API_KEY: "{{ FORUM_API_KEY }}"
   SEARCH_SERVER: "{{ FORUM_ELASTICSEARCH_URL }}"
+  SEARCH_SERVER_ES7: "{{ FORUM_ELASTICSEARCH_URL_ES7 }}"
```
Comment (davidjoy) [inline playbooks/roles/forum/defaults/main.yml:66]: I think `SEARCH_SERVER_ES7` may need to be added to `devstack_forum_env` section as well, but with the `http://edx.devstack.elasticsearch7:9200/` URL instead.

**Review by davidjoy — CHANGES_REQUESTED** [review]

File: playbooks/roles/forum/defaults/main.yml (hunk 1 above)
Comment (dianakhuang) [reply playbooks/roles/forum/defaults/main.yml:66]: Thanks for catching that. Updated.

File: playbooks/roles/forum/defaults/main.yml [hunk 2]
```diff
@@ -35,6 +35,8 @@ FORUM_ELASTICSEARCH_PORT: "9200"
 FORUM_ELASTICSEARCH_PROTOCOL: "http"
 FORUM_ELASTICSEARCH_URL: "{{ FORUM_ELASTICSEARCH_PROTOCOL }}://{{ FORUM_ELASTICSEARCH_HOST }}:{{ FORUM_ELASTICSEARCH_PORT }}"
 
+FORUM_ELASTICSEARCH_URL_ES7: "http://localhost:9200"
```
Comment (robrap) [inline playbooks/roles/forum/defaults/main.yml]: @dianakhuang: Is there a reason we collapsed host/port/protocol to URL? Note that we don't have a test here to match the old ES: https://github.com/edx/configuration/blob/b314385f3e468d62c05f278507138ef8cbc2ff84/playbooks/roles/forum/defaults/main.yml#L106-L107  Maybe this should be `FORUM_ELASTICSEARCH_HOST_ES7` instead?

Comment (dianakhuang) [reply playbooks/roles/forum/defaults/main.yml]: I thought it was unnecessarily complex to separate it out, but if we need to do that, we can do that.

File: playbooks/roles/forum/defaults/main.yml [hunk 3]
```diff
@@ -101,3 +105,4 @@ FORUM_VERSION: "master"
 #
 forum_services:
   - {service: "elasticsearch", host: "{{ FORUM_ELASTICSEARCH_HOST }}", port: "{{ FORUM_ELASTICSEARCH_PORT }}"}
+  - {service: "elasticsearch7", host: "{{ FORUM_ELASTICSEARCH_HOST_ES7 }}", port: "{{ FORUM_ELASTICSEARCH_PORT }}"}
```
Comment (robrap) [inline playbooks/roles/forum/defaults/main.yml:108]: @feanil: Does this service name need to match something, and could this cause an issue?

Comment (feanil) [reply playbooks/roles/forum/defaults/main.yml:108]: It's only used to test connectivity to that service, so it should be safe to add a second item.

https://github.com/edx/configuration/blob/master/playbooks/roles/forum/tasks/test.yml#L7

Comment (robrap) [reply playbooks/roles/forum/defaults/main.yml:108]: Great.  Thanks.

**Review by robrap — APPROVED** [review]
@dianakhuang: It probably makes sense for you to give a thumb on this comment, or for someone else to approve my change.  I will add comments to the other PRs to update to use `FORUM_ELASTICSEARCH_HOST_ES7`.

**Comment by dianakhuang** [comment]
👍 from me on these changes

**Review by robrap — APPROVED** [review]

**Review by davidjoy — APPROVED** [review]

**Pull request merged**